### PR TITLE
fix(slider-rtl): added rtl support for slider

### PR DESCRIPTION
**Pull request by EladBezalel**
- added bidi-property function to set a property by direction
- fixed slider in rtl

### CODE REVIEW

File: src/components/slider/slider.js [hunk 1]
```diff
@@ -549,6 +555,10 @@ function SliderDirective($$rAF, $window, $mdAria, $mdUtil, $mdConstant, $mdThemi
       var size = vertical ? sliderDimensions.height : sliderDimensions.width;
       var calc = (position - offset) / size;
 
+      if (!vertical && $mdUtil.bidi() === 'rtl') {
```
Comment (DevVersion) [inline src/components/slider/slider.js]: Since when is `$mdUtil.bidi()` returning a value?

Comment (EladBezalel) [inline src/components/slider/slider.js]: [util.js#L58](https://github.com/angular/material/blob/master/src/core/util/util.js#L58)

Comment (DevVersion) [inline src/components/slider/slider.js]: Oh, didn't saw that, thanks.

**Comment by ThomasBurleson** [comment]
@EladBezalel  this does not work properly!

screen shot 2016-03-09 at 1 49 32 pm


screen shot 2016-03-09 at 1 49 54 pm

**Comment by ThomasBurleson** [comment]
@EladBezalel - what is the status of this PR ?

**Comment by ThomasBurleson** [comment]
@EladBezalel - please rebase and confirm PR is valid.

**Comment by ThomasBurleson** [comment]
@EladBezalel - ping

**Comment by EladBezalel** [comment]
@ThomasBurleson working on it

**Comment by ThomasBurleson** [comment]
@EladBezalel  - status ping ?

**Comment by EladBezalel** [comment]
@ThomasBurleson this is ready IMO

**Pull request closed without merging**